api/routes.js

```diff
@@ -19,7 +19,27 @@ var routes = [
     },
     {
         method: 'GET',
-        path:'/login',
+        path:'/labels',
+        handler: require("./handlers/LabelsCtrl.js").get
+    },
+    {
+        method: 'POST',
+        path:'/labels',
+        handler: require("./handlers/LabelsCtrl.js").create
+    },
+    {
+        method: 'PUT',
+        path:'/labels',
+        handler: require("./handlers/LabelsCtrl.js").update
+    },
+    {
+        method: 'DELETE',
+        path:'/labels/{name}',
+        handler: require("./handlers/LabelsCtrl.js").remove
+    },
+    {
+        method: 'GET',
+        path:'/login', 
```

Comment:
If I was being picky. I would organise the routes in the order the person using the app will encounter them.
So that the routes.js file gives new developers an insight into the *journey* people have while using the app.
However, we will have test for that, so alphabetical is going to be good. :+1: